test/hotspot/jtreg/gc/stress/TestStressG1Humongous.java

```diff
@@ -23,15 +23,42 @@
 
 package gc.stress;
 
-/*
- * @test TestStressG1Humongous
+ /*
```

Comment:
Here and later, this is misaligned. All stars should align, like before.

Reply:
Stars ⭐  are aligned now, thanks!